loaders/include.js

```diff
@@ -3,11 +3,6 @@ const loaderUtils = require('loader-utils');
 module.exports.pitch = function pitch(remainingRequest) {
   const { globals = undefined, pre = [], post = [] } = loaderUtils.getOptions(this) || {};
 
-  // HACK: NamedModulesPlugin overwrites existing modules when requesting the same module via
-  // different loaders, so we need to circumvent this by appending a suffix to make the name unique
-  // See https://github.com/webpack/webpack/issues/4613#issuecomment-325178346 for details
-  this._module.userRequest = `include-loader!${this._module.userRequest}`;
```

Comment:
Will this break the v1 after removal? Probably we can check `this._module` and `this._moduoe.userRequest` existence, if they exist then we do this hack.